docs/css-style-guide.md

```diff
@@ -1,20 +1,29 @@
 # Hypothesis CSS Style Guide
 
-To enable efficient and safe evolution of our front end projects, we follow a few basic principles for styling:
+To enable efficient and safe evolution of our front end projects, we follow a few basic guides for styling:
```

Comment:
Using the term "guides" here per discussion last week, as "principles" are slightly higher-level in that model.

Reply:
"guides" or "guidelines"?